hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -195,11 +198,16 @@ module top_${top["name"]} #(
   end
 % endif
 
-  // clock assignments
+  // Clock assignments
```

Comment:
would you mind leaving a `TODO` here? I think what I originally intended was to surface the clock_names all the way up as ports, and connected them at the hierarchy above.  Probably about time to do that now...

Reply:
I think an issue would be better to track this than a `TODO`. I have created one here: #1391